src/main/webapp/search-script.js

```diff
@@ -23,8 +23,7 @@ async function showCourses() {
   url.searchParams.set("profName", profName);
   url.searchParams.set("units", units);
   url.searchParams.set("term", termName);
-  url.searchParams.set("schoolName", school);
-
+  url.searchParams.set("school-name", school);
```

Comment:
Why is this hyphenated but "profName" above isn't?

Reply:
Changed this to be hyphenated throughout